docs/applications/onepassword-connect/add-new-connect-server.rst

```diff
@@ -0,0 +1,84 @@
+##################################
+Add a new 1Password Connect server
+##################################
+
+This document describes how to set up a new 1Password Connect server.
+
+This is normally not required for SQuaRE-run environments, since all SQuaRE-run environments use the two already-existing 1Password Connect servers.
+The one in the :px-env:`roundtable-dev <roundtable-dev>` environment serves the vaults for development environments, and one in the :px-env:`roundtable-prod <roundtable-prod>` environment serves the vaults for production environemnts.
+However, these instructions may be helpful for other Phalanx users who want to st up an independent 1Password Connect server, or if we need to add a new server for some reason.
+
+When following these instructions, you will be creating a new `Secrets Automation workflow <https://developer.1password.com/docs/connect/get-started/>`__.
+You will need to have permissions to create that workflow for the vault for your environment.
+
+Create the workflow
+===================
+
+In the following steps, you will create a 1Password Secrets Automation workflow for the 1Password vault for your environment, and save the necessary secrets to another 1Password vault.
+
+#. Log on to the 1Password UI via a web browser.
+
+#. Click on :menuselection:`Integrations` in the right sidebar under **LSST IT**.
+
+#. Click on the :guilabel:`Directory` tab at the top of the screen.
+
+#. Under :guilabel:`Infrastructure Secrets Management` click on :guilabel:`Other`.
+
+#. Click on :guilabel:`Create a Connect server`.
+
+#. Under :guilabel:`Environment Name`, enter :samp:`RSP {environment}` where *environment* is the Phalanx environment in which this 1Password Connect server will be running (**not** the vaults that it will serve).
+   Then, click :guilabel:`Choose Vaults` and select the vaults that should be accessible through this 1Password Connect server.
+   Click :guilabel:`Add Enviroment` to continue.
+
+#. Next, 1Password wants you to create an access token for at least one environment.
+   This is the token that will be used by the Phalanx command-line tool to access secrets for that environment.
+   It will have access to one and only one vault.
```

Comment:
```suggestion
   It will have access to one and only one 1Password vault.
```


maybe we can always qualify "vault" as being a "1Password vault" to avoid confusion.